include/RAJA/policy/cuda/policy.hpp

```diff
@@ -196,17 +196,27 @@ static_assert(MAX_BLOCK_SIZE % WARP_SIZE == 0,
               "RAJA Assumption Broken: MAX_BLOCK_SIZE not "
               "a multiple of WARP_SIZE");
 
+struct cuda_synchronize
+    : make_policy_pattern_launch_t<Policy::cuda, Pattern::synchronize, Launch::sync> {
+};
+
 }  // end namespace cuda
 }  // end namespace policy
 
 using policy::cuda::cuda_exec;
+
+template<size_t BLOCK_SIZE>
+using cuda_exec_async = policy::cuda::cuda_exec<BLOCK_SIZE, true>;
```

Comment:
@trws @ajkunen I added an alias for `cuda_exec_async` also. The boolean argument is a bit opaque.